react-context-why-when-how/README.md

```diff
@@ -0,0 +1,7 @@
+# Exercises
+
+| Exercise | Problem | Solution |
+| --- | --- | --- |
+| Demo | [Demo](https://codesandbox.io/s/github/bitovi/trainings/tree/main/Demo) |  |
```

Comment:
I think this should be https://codesandbox.io/s/github/bitovi/trainings/tree/main/react-context-why-when-how/Demo